packages/collector/test/tracing/messaging/bull/util.js

```diff
@@ -126,18 +128,19 @@ exports.buildReceiver = function (queue, processType, log, jobName, isConcurrent
       throw new Error(`Option ${processType} is invalid`);
   }
 
-  if (jobName && isConcurrent) {
-    log(`Job named ${jobName} and concurrent`);
-    currentTypeArgs.unshift(jobName, NUMBER_OF_PROCESSES);
-  } else if (jobName && !isConcurrent) {
+  // You cannot set name + concurrency at the same time. The concurrency is ignored in that case.
```

Comment:
All tests used `BULL_CONCURRENCY_ENABLED=true` but it had no effect because of ^ 😭